instrumentation/logback/logback-mdc-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/mdc/v1_0/OpenTelemetryAppender.java

```diff
@@ -30,6 +27,9 @@
 public class OpenTelemetryAppender extends UnsynchronizedAppenderBase<ILoggingEvent>
     implements AppenderAttachable<ILoggingEvent> {
   private boolean addBaggage;
+  private String traceIdKey;
```

Comment:
if you'd set `traceIdKey = LoggingContextConstants.TRACE_ID` here you wouldn't need to do the null checks below